os/dbuild.sh

```diff
@@ -113,6 +113,7 @@ function SELECT_OPTION()
 				echo "  \"d. Download\""
```

Comment:
Please change "The toolchaing commands" to "The toolchain commands" in the commit message